tests/boost/test_prometheus_metrics.cpp

```diff
@@ -5,6 +5,7 @@
 #include <boost/test/included/unit_test.hpp> // for BOOST_PP_IIF_1, BOOST_CHECK, BOOST_PP_BOOL_2
 #include <iostream>                          // for cout, ostream
 #include <string>                            // for string, allocator, basic_string, operator==
+#include <cmath>                             // for sqrt
```

Comment:
Right, this is jsut `#include <cmath>` And then you might have to refer to `sqrt` as `std::sqrt`. (Check the build to verify.) See https://en.cppreference.com/w/cpp/numeric/math/sqrt.

Reply:
It also will not let me do this =)

```
../../../tests/boost/test_prometheus_metrics.cpp:37:17: error: division by zero [-Werror=div-by-zero]
     bazinf.set(1/0);
```